src/component/shared/nav/Nav.jsx

```diff
@@ -1,10 +1,19 @@
-import { GiHamburgerMenu } from "react-icons/gi";
 import { Link } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import { GiHamburgerMenu } from "react-icons/gi";
+import { toggleSidebar } from "../../../features/displaySlice";
+import { logOut } from "../../../features/authSlice";
```

Comment:
barrel export

Reply:
yes, will implement in next PR for sure !